### PR TITLE
Add git stash question to Git Quiz

### PR DESCRIPTION
**Pull request by stmapman7**
## Summary of changes

I added a question concerning the pros and cons of using the git stash command.

## Checklist

- [X] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [X] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [X] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [X] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [X] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -467,7 +467,7 @@ const gitQuiz = [
       "Git is a popular version control system and GitHub is a hosting service for Git repositories.",
     Distractor1:
       "GitHub is a type of version control system, while Git is a hosting service for Git repositories.",
-    Distractor2: "Git and GitHub are the same thing",
+    Distractor2: "Git and GitHub are the same things",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:470]: this change is extraneous for this PR, please remove it

File: src/data/git-quiz.ts [hunk 2]
```diff
@@ -477,7 +477,7 @@ const gitQuiz = [
   {
     Question: "In Git, how do you apply a stash if you have multiple?",
     Answer: "git stash pop stash@{x}",
-    Distractor1: "git stash pop",
+    Distractor1: "Git stash pop",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:480]: this change is extraneous for this PR, please remove it

File: src/data/git-quiz.ts [hunk 3]
```diff
@@ -486,7 +486,7 @@ const gitQuiz = [
   },
   {
     Question: "In Git, how do you see a summary for a single stash?",
-    Answer: "git stash show",
+    Answer: "Git stash show",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:489]: this change is extraneous for this PR, please remove it

File: src/data/git-quiz.ts [hunk 4]
```diff
@@ -542,13 +542,13 @@ const gitQuiz = [
     Distractor2: "Nano",
     Distractor3: "VS Code",
     Explanation:
-      "Vim stands for Vi Improved and it is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
+      "Vim stands for Vi Improved and is a terminal-based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:545]: this change is extraneous for this PR, please remove it

File: src/data/git-quiz.ts [hunk 5]
```diff
@@ -542,13 +542,13 @@ const gitQuiz = [
...
     Link: "https://www.freecodecamp.org/news/learn-vim-beginners-tutorial/"
   },
   {
     Question: "How do you unstage a staged file in git?",
     Answer: "git restore --staged <file-path>",
-    Distractor1: "git unstage <file-path>",
+    Distractor1: "git unstaged <file-path>",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:551]: this change is extraneous for this PR, please remove it

File: src/data/git-quiz.ts [hunk 6]
```diff
@@ -630,6 +630,17 @@ const gitQuiz = [
     Explanation:
       "The 'git help' command will display help information on common Git commands. The 'git help git' command will provide a complete overview of the system.",
       Link: "https://www.freecodecamp.org/news/best-git-tutorial/"
+},
+  {
+    Question: "When using the 'git stash' command, what is one of the major disadvantages of using it?",
+    Answer: 
+      "cluster and confusion",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:637]: this is the only choice between distractors and answers that is not a full sentence, answers and distractors for a single question should all have the same style please

File: src/data/git-quiz.ts (hunk 1 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Removed.

File: src/data/git-quiz.ts (hunk 2 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Removed.

File: src/data/git-quiz.ts (hunk 3 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Removed

File: src/data/git-quiz.ts (hunk 5 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Removed

File: src/data/git-quiz.ts (hunk 6 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Changed the answer to a complete sentence.

File: src/data/git-quiz.ts (hunk 4 above)
Comment (stmapman7) [reply src/data/git-quiz.ts]: Removed

File: src/data/git-quiz.ts [hunk 7]
```diff
@@ -542,7 +542,7 @@ const gitQuiz = [
     Distractor2: "Nano",
     Distractor3: "VS Code",
     Explanation:
-      "Vim stands for Vi Improved and it is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
+      "Vim stands for Vi Improved and is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
```
Comment (ilenia-magoni) [inline src/data/git-quiz.ts:545]: there is still this extraneous change

Comment (stmapman7) [reply src/data/git-quiz.ts]: Fixed.

**Review by stmapman7 — COMMENTED** [review]
I fixed the formatting (there was an extra space).

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**